src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -495,6 +495,12 @@
   <data name="SecondsAgo" xml:space="preserve">
     <value>{0} seconds ago</value>
   </data>
+  <data name="SecondAgo" xml:space="preserve">
+    <value>{0} second ago</value>
```

Comment:
Shouldn't this be hardcoded to 1 second ago?

Reply:
'1 day ago', '1 hour ago' and '1 minute ago' are not hardcoded either.
Should all of these be hardcoded?

Reply:
Probably

Reply:
Hmm, but hardcoding would require all other language strings to be changed as well.

Reply:
That's true, it's not a major concern to update the other strings.

Reply:
Okey, so I will hardcode '1 second ago' only.